ui/app/(ai)/analyst/(tools)/checks.ts

```diff
@@ -0,0 +1,18 @@
+import { tool } from "@langchain/core/tools";
+
+import { aiGetProviderChecks } from "@/lib/lighthouse/helperChecks";
+import { checkSchema } from "@/types/ai/checks";
+
+export const getProviderChecksTool = tool(
+  async ({ provider_type }) => {
+    console.log("=> Invoking getProviderChecksTool - ", provider_type);
```

Comment:
Is this console log needed? If not, can we remove it, please?

Reply:
Will remove all console.log statements in tools